### ISSUE TITLE
days_from_uploading_to_hiding must be a positive integer

### ISSUE THREAD

**Issue by berbiche** [Backblaze/terraform-provider-b2#4]
Hi, according to the [official documentation](https://www.backblaze.com/b2/docs/lifecycle_rules.html), I should be able to define a `lifecycle_rules` with a value of null for `daysFromUploadingToHiding` to disable it.

Yet, when I set it to null or omit it from my terraform configuration, I get the error `daysFromUploadingToHiding must be positive (bad_request)`.

System info:
``` console
$ terraform --version
Terraform v0.14.5
+ provider registry.terraform.io/backblaze/b2 v0.2.0
+ provider registry.terraform.io/carlpett/sops v0.6.0
+ provider registry.terraform.io/cloudflare/cloudflare v2.18.0
+ provider registry.terraform.io/ovh/ovh v0.10.0
```

**Comment by mlech-reef** [comment]
What is your `lifecycle_rules` configuration? You can not set both to null for example:

> Setting both to null is not allowed, because the rule would do nothing.

Or set the other one to zero:

> Setting either daysFromUploadingToHiding or daysFromHidingToDeleting to 0 is not allowed. When either is set, it must be a positive number.

Bet regards,
Maciej Lech

**Comment by berbiche** [comment]
My `lifecycle_rules` block is defined as

``` terraform
lifecycle_rules {
  file_name_prefix = ""
  days_from_hiding_to_deleting = 30
}
```

**Comment by ppolewicz** [comment]
Can we add a `--verbose` option that would produce debug logs as in b2cli?

**Comment by mlech-reef** [comment]
@berbiche I will look into this.

@ppolewicz, terraform has its own debugging mechanism and unfortunately, it's not easy to fetch the logs from SDK. We may want to add logging to the file according to the ``TF_LOG`` env. I will check it.

**Comment by ppolewicz** [comment]
can't the provider just use stdout/stderr?

**Comment by mlech-reef** [comment]
> can't the provider just use stdout/stderr?
No

**Comment by mlech-reef** [comment]
@berbiche,

The problem has been fixed in [v0.2.1](https://github.com/Backblaze/terraform-provider-b2/releases/tag/v0.2.1).

Let us know if it works for you.

Best regards,
Maciej Lech

**Comment by berbiche** [comment]
I confirm the bug is resolved. Thanks for the quick fix! :smile:
